go/src/koding/socialapi/presence/presence.go

```diff
@@ -0,0 +1,38 @@
+package presence
+
+import (
+	"errors"
+	"net/http"
+	"net/url"
+
+	"koding/socialapi"
+)
+
+type Client struct {
```

Comment:
why do you need to refactor? this package has the exact same functionality. Why do you need to access to http.Request?